sideboard/run_debug_server.py

```diff
@@ -6,6 +6,10 @@
 import sideboard.server
 
 if __name__ == '__main__':
+    # import pydevd
+    # print("running debug server2...", flush=True)
+    # pydevd.settrace('10.0.0.29', port=5000, stdoutToServer=True, stderrToServer=True)
+
     debugger_helpers_all_init()
```

Comment:
Should we put these TODOs inside the code so they're more accessible to devs/future us?